### ISSUE TITLE
Lay On Hands

### ISSUE THREAD

**Issue by Nostealth** [dalaranwow/dalaran-wow#5169]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
If a paladin is affected by heal reduction ( wound poison/aimed shot/ mortal strike), Lay on Hands should heal half of the amount.
**Expected behaviour**: Tell us what should happen instead
It doesn't matter what healing reduction you have on the target paladin will always heal to full HP.
**Steps to reproduce the problem**:

1. Duel a paladin and take it to low hp.
2. Ask him to use lay on hands while affected by wound poison.
3. See results.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
https://cdn.discordapp.com/attachments/363382145303183361/388845078472425475/WoWScrnShot_120917_021042.jpg

**Comment by Zarjofgt** [comment]
Are you sure its not just healing so much that half of it is still 100% of his hp? With spellpower and or crit

**Comment by Aridam** [comment]
It's supposed to be reduced in correlation to the potency of the Healing Reduction effect indeed (for example, Mortal Strike: -50% to received Healing; Improved Mind Blast: -20% to received Healing, etc.), but do take into consideration what was said just-above in that if one's HP is so big and the Lay itself managed to be a Crit, it could've been that it still managed to heal up to full Health even _with_ a Healing Reduction effect present.

In a possible situation someone could have 25k HP and proceed to use Lay on someone with 12k of a Health Pool -- even if the receiver is affected by Mortal Strike, Wound Poison, 2x Furious Attacks, and so on, the halved _non-Crit_ amount with any of those Debuffs would still be as much as their entire Health nonetheless.

**Comment by Nostealth** [comment]
There is a print screen attached to this thread. It states simply that a retribution paladin with 27964 hp has managed to heal 23007(4957Overheal)..as a non-crit. If you sum up the heal and the overheal it will result in the paladin full health, while being affected by wound poison. Not sure what's so difficult to understand?!

**Comment by Zarjofgt** [comment]
how about a screenshot with and without wound poison?

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Nostealth,
Issue confirmed on PTR.
Regardless if the player has any -% healing debuff on himself or target (if he wants to heal someone else) the result is always 100% of paladin's hp in healing and 150% if it is a critical.
![image](https://user-images.githubusercontent.com/6797611/33993712-66493598-e0e0-11e7-8407-299d28fad353.png)

**Comment by TeridanDW** [comment]
Works as intended on PTR.

**Comment by Aridam** [comment]
This is still an issue on Live -- Paladins abusing this bug to fully-heal with LoH on anyone despite of them having Healing-reducing effects.

**Comment by Aridam** [comment]
Fix this. People keep abusing this. On purpose. The whole time. It's gamebreaking.

**Comment by robinsch** [comment]
I can't do stuff. @Lothloryen I fixed it.
